hram.asm

```diff
@@ -125,8 +125,17 @@ hPrintNumDivisor:: ds 3 ; ffb7 big-endian
 hPrintNumTemp:: ds 3 ; ffba big-endian
 
 ENDU
-    ds 19 ; TODO
 
+	ds 10 ; TODO
+	
+hffc7:: db ; ffc7
```

Comment:
Unknown HRAM labels are in all caps (`hFFC7`), but it really doesn't matter either way.